frontend/src/components/NormReferences.vue

```diff
@@ -23,23 +24,40 @@ const norms = computed({
   },
 })
 
+function hasSingleNorms(normEntry: NormReference) {
+  if (normEntry.singleNorms)
+    return (
+      normEntry.singleNorms?.length > 1 || !normEntry.singleNorms[0].isEmpty
```

Comment:
When length is 0 this line will throw an TypeError: Cannot read properties of undefined (reading 'isEmpty').
Use case: Use deletes all singleNorms.

Reply:
Gut gesehen! hab ich korrigiert

Reply:
But is normEntity.singleNorms not false if it's a empty array?

Reply:
Hm, I don't see your fix 🤔 ? Did everything work fine git-wise?

Reply:
This condition seems to work though :)